MekHQ/src/mekhq/campaign/force/ForceType.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (c) 2025 - The MegaMek Team. All Rights Reserved.
+ *
+ * This file is part of MekHQ.
+ *
+ * MekHQ is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MekHQ is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MekHQ. If not, see <http://www.gnu.org/licenses/>.
+ */
+package mekhq.campaign.force;
+
+import megamek.logging.MMLogger;
+
+/**
+ * Represents the various types of forces available.
+ *
+ * It is used to classify and manipulate forces within the game.
+ */
+public enum ForceType {
+    // region Enum Declarations
+    /**
+     * Standard force type, typically used for combat and general operations.
+     */
+    STANDARD("Standard"),
+
+    /**
+     * Support force type, generally ignored by MekHQ.
+     */
+    SUPPORT("Support"),
+
+    /**
+     * Convoy force type, typically used for transport and supply operations.
+     */
+    CONVOY("Convoy"),
+
+    /**
+     * Security force type, typically used for protection and guarding operations.
+     */
+    SECURITY("Security");
+
+
+    // Fields
+    private final String name;
```

Comment:
redundant, ForceType.STANDARD.name() would return `STANDARD`, while `.getName()` returns `Standard`, if you want to use it in text you should use i18n, so neither use would be good enough here